includes/admin/beta-features.php

```diff
@@ -0,0 +1,316 @@
+<?php // phpcs:disable Universal.Files.SeparateFunctionsFromOO.Mixed
+/**
+ * Admin Beta Features
+ *
+ * This file contains the admin beta features functionality for Secure Custom Fields.
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.3
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( ! class_exists( 'SCF_Admin_Beta_Features' ) ) :
+	/**
+	 * Class SCF_Admin_Beta_Features
+	 *
+	 * This class provides different beta features that eventually will land on secure custom fields.
+	 */
+	class SCF_Admin_Beta_Features {
+
+		/**
+		 * Contains an array of admin beta feature instances.
+		 *
+		 * @var array
+		 */
+		private $beta_features = array();
+
+		/**
+		 * The active beta feature.
```

Comment:
What does this mean? Just by looking at the declaration and comment, before checking the rest of the code, it sounds like only one beta feature can be active at a time.